xapian-applications/omega/omega.cc

```diff
@@ -47,6 +47,7 @@
 #include "str.h"
 #include "stringutils.h"
 #include "expand.h"
+#include "common/parseint.h"
```

Comment:
`common` is on the header search path, so `common/` isn't needed here (several of the headers above are in there too).

Reply:
thanks i will fix that.